buildbot.mariadb.org/master-libvirt/master.cfg

```diff
@@ -235,88 +187,101 @@ f_rpm_upgrade.addStep(getScript('rpm-upgrade.sh'))
+    elif platform == 'aarch64':
+        libvirt_worker_name = 'bb-eq-arm1-' + os_name + '-' + os_version
+        connection_url = 'qemu+ssh://buildbot@100.64.100.17/system'
+
+    image_path = '/var/libvirt/images/' + libvirt_worker_name
+
+    c['workers'].append(worker.LibVirtWorker(libvirt_worker_name,
+        config["private"]["worker_pass"]["libvirt"],
+        util.Connection(connection_url),
+        image_path, build_wait_timeout=0, max_builds=1))
+
+    if builder_type == 'deb':
+        factory_install = f_deb_install
+        factory_upgrade = f_deb_upgrade
+        build_arch = platform
+    elif builder_type == 'rpm':
+        factory_install = f_rpm_install
+        factory_upgrade = f_rpm_upgrade
+        build_arch = os_name + os_version + '-' + platform
+
+    c['builders'].append(
+        util.BuilderConfig(name=builder_name,
+            workernames=libvirt_worker_name,
+            tags=[os_name, builder_type, 'install', 'kvm'],
+            collapseRequests=True,
+            nextBuild=nextBuild,
+            canStartBuild=canStartBuild,
+            properties={
+                'systemdCapability': 'yes',
+                'needsGalera': 'yes',
+                'dist_name': os_name,
+                'version_name': os_info[os_info_name]['version_name'],
+                'arch': platform},
+            factory=factory_install))
+
+    # Add major upgrade builder
+    major_upgrade_name = '-'.join(builder_name.split('-')[:5]) + '-major-upgrade'
+
+    c['builders'].append(
+        util.BuilderConfig(name=major_upgrade_name,
+          workernames=libvirt_worker_name,
+          tags=[os_name, builder_type, 'upgrade', 'kvm'],
+          collapseRequests=True,
+          nextBuild=nextBuild,
+          canStartBuild=canStartBuild,
+          properties={
+              'systemdCapability': 'yes', 
+              'needsGalera': 'yes', 
+              'dist_name': os_name, 
+              'version_name': os_info[os_info_name]['version_name'],
+              'arch': platform, 
+              'test_mode': 'server', 
+              "test_type": "major"
+             },
+          factory=factory_upgrade))
+
+    # Add minor upgrade builder
+    minor_upgrade_name = '-'.join(builder_name.split('-')[:5]) + '-minor-upgrade'
+
+    c['builders'].append(
+        util.BuilderConfig(name=minor_upgrade_name,
+          workernames=libvirt_worker_name,
+          tags=[os_name, builder_type, 'upgrade', 'kvm'],
+          collapseRequests=True,
+          nextBuild=nextBuild,
+          canStartBuild=canStartBuild,
+          properties={
+              'systemdCapability': 'yes', 
+              'needsGalera': 'yes', 
+              'dist_name': os_name, 
+              'version_name': os_info[os_info_name]['version_name'],
+              'arch': platform, 
+              'test_mode': 'server', 
+              "test_type": "minor"},
+          factory=factory_upgrade))
```

Comment:
About properties:
- we should check that they are actually necessary (not sure about all).
- can you also define a `BB_CI=true` please?

Reply:
Yep, if anything changed in the scripts, probably some of the need to be removed. We should sync on this

Reply:
@fauust can you confirm that only 'arch' is needed for the scripts? This is what it appears to me!

Reply:
Let me double check that tomorrow and get back to you